README.md

```diff
@@ -75,29 +75,67 @@ Theatre assumes developers have several tools installed to provide development
 and testing capabilities. The following will configure a macOS environment with
 all the necessary dependencies:
 
+1. Install Docker Desktop for macOS from https://www.docker.com/get-started
+
+2. Install Go, kubectl, kustomize, and kind via Homebrew
 ```shell
-brew cask install docker
-brew install go kubernetes-cli kustomize
-curl -fsL -o /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.8.1/kind-darwin-amd64 \
-  && chmod a+x /usr/local/bin/kind
-mkdir /usr/local/kubebuilder
-curl -fsL https://go.kubebuilder.io/dl/2.3.1/$(go env GOOS)/$(go env GOARCH) | tar -xvz --strip=1 -C /usr/local/kubebuilder
-export KUBEBUILDER_ASSETS=/usr/local/kubebuilder/bin
+brew install go kubernetes-cli kustomize kind
 ```
 
-Running `make` should now compile binaries into `bin`.
+Note: kubectl is already included with Docker Desktop, so you might get a message
+about the command getting installed by Homebrew but not linked.
+
+3. (Optional) Download and install Kubebuilder
+
+This step is not required for running acceptance tests. It is only required
+for writing new Kubernetes operators.
+
+We are using an older version of Kubebuilder, which doesn't have pre-built
+binaries for Apple silicon. Therefore, if your workstation is an ARM64 Mac
+then you can either run the x86 binaries (which will run in emulation mode on
+Rosetta) or you can build Kubebuidler from sources.
+
+- Option A: pre-built binaries (for Intel Macs or in emulation mode on M1 Macs)
+```shell
+sudo mkdir /usr/local/kubebuilder
+curl -fLs https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_darwin_amd64.tar.gz | \
+  sudo tar x --strip=1 -C /usr/local/kubebuilder
+```
+
+- Option B: build form source (suitable for both Intel and M1 Macs)
```

Comment:
I'm not sure if this will work. The pre-built release includes `kubebuilder` as well as three other executables: 
`etcd`, `kube-apiserver`, `kubectl`. Buiding from source only creates a `kubebuilder` executable.

Reply:
Yep, this won't work. `make test` will fail without these files.